server/bundler/loader.js

```diff
@@ -44,7 +44,17 @@ function getSectionsModule( sections ) {
 			'	load: function() {',
 			'		' + sectionLoaders,
 			'	}',
-			'};'
+			'};',
+			'function loadCSS( section ) {',
+			'	if ( section.css ) {',
+			'		var link = document.createElement( "link" );',
```

Comment:
We should probably unload the css of the current section before switching to the new one. Using an `id` property (like `id="section-css"`) and checking if it exists before adding it again could be an easy fix.

Reply:
Why do we need to unload?